challenges-46-50.yml

```diff
@@ -0,0 +1,219 @@
+manual_stream:
+  question: |
+    How does the function `renderStreamMessage` work?
+  explanation: |
+    `renderStreamMessage` allows us to modify different areas on a page
+    using the Turbo Streams markup
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          `renderStreamMessage` takes a Response object from a `fetch` function and
+          processes the response in order to make changes in your page.
+        b: |
+          `renderStreamMessage` takes a plain string as an argument and uses your pages'
```

Comment:
```suggestion
          `renderStreamMessage` takes a plain string as an argument and uses your page's
```

The apostrophe location seemed deliberate, but I think we're only talking about one page here (whatever one page the user is viewing).

Reply:
You are correct!